website/content/docs/enterprise/entropy-augmentation.mdx

```diff
@@ -47,6 +47,8 @@ and include the following:
 - [PKI](/docs/secrets/pki) issuer key generation, but not for leaf certificate private keys
 - [`/sys/tools/random`](/api-docs/system/tools#generate-random-bytes) endpoint (Vault 1.11+)
 - [SSH](/docs/secrets/ssh) CA key generation, but not for key pair generation
+- [KMIP](/docs/secrets/kmip) uses EA for its TLS CA, server, and client
```

Comment:
Well, don't forget the "managed objects", which are the keys and secret data objects that KMIP manages for clients. I think this means that all storage in KMIP is sealed wrapped, but I am not familiar enough with the storage code to verify that that is true.